packages/service-core/src/sync/sync.ts

```diff
@@ -198,7 +207,7 @@ async function* streamResponseInner(
           user_id: syncParams.user_id,
           // Passing undefined will emit a full sync complete message at the end. If we pass a priority, we'll emit a partial
           // sync complete message.
-          forPriority: !isLast ? priority : undefined
+          forPriority: !isLast ? priority : null
```

Comment:
The comment above mentions "Passing undefined will...". I assume that also applies to `null`?

Reply:
It does, I've updated the comment.